src/test/FreeC/Pass/CompletePatternPassTests.hs

```diff
@@ -0,0 +1,172 @@
+-- name subject to change
+
+-- | This module contains tests for "FreeC.Pass.CompletePatternPassTests".
+
+module FreeC.Pass.CompletePatternPassTests where
+
+import           Test.Hspec
+
+import           FreeC.Monad.Class.Testable
+import           FreeC.Pass.CompletePatternPass
+import           FreeC.Test.Environment
+import           FreeC.Test.Parser
+-------------------------------------------------------------------------------
+-- Tests                                                                     --
+-------------------------------------------------------------------------------
+
+testCompletePatternPass :: SpecWith ()
+testCompletePatternPass = describe "FreeC.Pass.CompletePatternPass" $ do
```

Comment:
You don't really need the heading if there is just a single function in your module. However, there should be documentation for the function itself.

```suggestion

-- | Test group for "FreeC.Pass.CompletePatternPass" tests.
testCompletePatternPass :: SpecWith ()
testCompletePatternPass = describe "FreeC.Pass.CompletePatternPass" $ do
```